tcp_protocol/Cargo.toml

```diff
@@ -6,4 +6,6 @@ edition = "2021"
 [dependencies]
 serde = { version = "1", features = ["alloc", "derive"], default_features = false }
 serde_derive = "1.0"
-serde_json = { version = "1", features = ["alloc"], default_features = false }
\ No newline at end of file
+serde_json = { version = "1", features = ["alloc"], default_features = false }
+
+kanata-parser = { path = "../parser" }
```

Comment:
Hm this dependency doesn't look quite right to me. I think it's better to put `simple_sexpr_to_json_array` into the kanata crate under `tcp_server`.

Reply:
> the kanata crate under `tcp_server`.

you mean `src/tcp_server.rs`? the file where `TcpServer` is defined? next to it?

Reply:
Yea correct, though it might also be fine to have it just as part of:
```
                        CustomAction::PushMessage(_message) => {
```

The important part being not having `tcp_protocol` take a dependency on the parser :)